### PR TITLE
feat: [#176176848] Added explanatory bottom sheet for in-app payments card section

### PR DESCRIPTION
**Pull request by DavideValdo**
https://user-images.githubusercontent.com/26572302/104206448-d5accd80-542f-11eb-9fb1-f50c4a4ef56c.mov

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -1061,6 +1061,9 @@ wallet:
         active: Active
         arriving: Arriving
         incompatible: Incompatible
+        bottomSheetTitle: What are in app payments?
```
Comment (thisisjp) [inline locales/en/index.yml:1064]:
```suggestion
        bottomSheetTitle: What does in-app payment mean?
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -1061,6 +1061,9 @@ wallet:
...
+        bottomSheetBody: IO lets you pay all pagoPA notices, even those on paper. If you wish, you can use this payment method. Anyhow, no in app payment will be triggered without your explicit consent.
```
Comment (thisisjp) [inline locales/en/index.yml:1065]:
```suggestion
        bottomSheetBody: With IO you can pay all pagoPA notices, even those on paper. When you perform in-app payments, you'll need to authorize the transaction using your biometrics or the unlock code.
```

File: locales/it/index.yml [hunk 3]
```diff
@@ -1086,6 +1086,9 @@ wallet:
         active: Attivo
         arriving: In arrivo
         incompatible: Non compatibile
+        bottomSheetTitle: Cosa sono i pagamenti in app?
+        bottomSheetBody: IO ti permette di pagare tutti gli avvisi pagoPA, anche quelli cartacei. Se lo desideri, puoi usare questo metodo di pagamento. In ogni caso, nessun pagamento in app verrà effettuato senza il tuo esplicito consenso.
```
Comment (thisisjp) [inline locales/it/index.yml:1090]:
```suggestion
        bottomSheetBody: IO ti permette di pagare tutti gli avvisi pagoPA, anche quelli cartacei. Quando effettui un pagamento in app, ti verrà chiesto di autorizzarlo tramite riconoscimento biometrico o codice di sblocco.
```

**Review by thisisjp — CHANGES_REQUESTED** [review]
I made some small copy changes :)

File: ts/features/wallet/component/PagoPaPaymentCapability.tsx [hunk 4]
```diff
@@ -83,28 +97,52 @@ const availabilityBadge = (badgeType: PaymentSupportStatus) => {
...
-          {I18n.t("wallet.methods.card.pagoPaCapability.title")}
-        </H4>
-        <H5
-          weight={"Regular"}
-          color={"bluegrey"}
-          testID={"PagoPaPaymentCapabilityDescription"}
-        >
-          {I18n.t("wallet.methods.card.pagoPaCapability.description")}
-        </H5>
+    <TouchableOpacity onPress={present}>
```
Comment (CrisTofani) [inline ts/features/wallet/component/PagoPaPaymentCapability.tsx:124]: imho in this case there's no need to use a component from 3rd party library, let's use `TouchableDefaultOpacity` from `ts/components/TouchableDefaultOpacity.tsx` which is based on `TouchableOpacity` from `react-native` package

**Review by CrisTofani — CHANGES_REQUESTED** [review]

**Review by CrisTofani — APPROVED** [review]

**Pull request merged**